masterlist.yaml

```diff
@@ -7084,12 +7099,43 @@ plugins:
       # version: 1.58
       - crc: 0xD7423A4E
         util: 'SSEEdit v4.0.0'
+  - name: 'imp_helm_Immersive_Armors.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
+        name: 'Improved Closedfaced Helmets Patches'
+    req:
+      - 'imp_helm_legend.esp'
```

Comment:
LOOT checks a plugins header for its masters and will sort a plugin after its master automatically.
So `afters` and `req` aren't required for any modules listed as a master in a plugin header.
You can view a plugins masters in Mod Organizer, Wrye Bash or SSEEdit.

![image](https://user-images.githubusercontent.com/1944639/57930873-fadde700-78ae-11e9-80a0-c3dec16f6059.png)

Reply:
Removed all the redundant dependencies.